doc/xml/user-guide.xml

```diff
@@ -2337,6 +2337,8 @@
 
             <p>A new host named <host>repository</host> is created to store the cluster backups.</p>
 
+            <admonition type="note">The <backrest/> version installed on the {[host-repo1]} host must exactly match the version installed on the <postgres/> host.</admonition>
```

Comment:
Replace `{[host-repo1]}`  with `<host>repository</host>` - In reading through this section of the document, repo1 is never mentioned except here whereas repository is, so repo1 vs repository would just confuse things.

Reply:
I had also come to that conclusion and still managed to get it wrong. Fixed now.